ding/policy/bc.py

```diff
@@ -104,7 +112,10 @@ def _forward_eval(self, data):
                 data = to_device(data, self._device)
             self._eval_model.eval()
             with torch.no_grad():
-                output = self._eval_model.forward(data)
+                if not self._cfg.continuous:
```

Comment:
why still some codes about metric evaluator here, you should modify it like this:
```python
if self._cfg.metric.eval_type == 'metric':
  ...
elif self._cfg.metric.eval_type == 'interaction':
  ...
else:
  raise KeyError
```